tgstation.dme

```diff
@@ -3875,7 +3875,6 @@
 #include "modular_skyrat\master_files\code\modules\events\spider_infestation.dm"
 #include "modular_skyrat\master_files\code\modules\jobs\departments\departments.dm"
 #include "modular_skyrat\master_files\code\modules\jobs\job_types\cyborg.dm"
-#include "modular_skyrat\master_files\code\modules\language\language_holders.dm"
```

Comment:
About this, we have a more populated file located here:
\modular_skyrat\modules\customization\modules\language\language_holder.dm